e2e/utils/touch.js

```diff
@@ -101,14 +101,24 @@ const touchCard = async(browser, cardSelector)=>{
 			await touch(el, y + 15);
 		};
 
-		const dragHandler = document.querySelector(`${cardSelectorBrowser} [data-testid=cardDragTarget]`);
+		const dragHandler = document.querySelector(`${cardSelectorBrowser}`);
 
 		await touchCardAt(dragHandler);
 	}, cardSelector);
 };
 
+const touchElementInsideCard = async(browser, cardSelector, childElementSelector)=>{
+	await browser.execute(async(cardSelectorBrowser, childElementSelectorBrowser) => {
+		const cardDocument = document
+			.querySelector(`${cardSelectorBrowser} article [data-testid="amp-document-single"]`);
```

Comment:
Is this selector valid for AMP and touch? (or the method only used in AMP?)

I would apply a generic selector or rename the method to make it explicitly useful for AMP tests only

Reply:
its only used in web actually. for amp it is not possible to select inside since the content of the card renders inside an iframe

Reply:
> its only used in web actually. for amp it is not possible to select inside since the content of the card renders inside an iframe

So the selector has nothing to do with amp even the data-attribute value contains the amp string?